include/odp/api/spec/traffic_mngr.h

```diff
@@ -1841,6 +1885,48 @@ typedef struct {
 int odp_tm_node_fanin_info(odp_tm_node_t             tm_node,
 			   odp_tm_node_fanin_info_t *info);
 
+/** The odp_tm_node_stats_t record type  is used to return various stats
+ * supported by a given tm_node via the odp_tm_node_stats() function.
+ */
+typedef struct odp_tm_node_stats_t {
+	/** Packets dropped by this node post scheduling/shaping at this node */
+	uint64_t dropped_pkts;
+
+	/** Octets dropped after scheduling/shaping at this node */
+	uint64_t dropped_octets;
+
+	/** Green packets that are sent through this tm node */
+	uint64_t green_pkts;
+
+	/** Green octets that are sent through this tm node */
+	uint64_t green_octets;
+
+	/** Yellow packets that are sent through this tm node */
+	uint64_t yellow_pkts;
+
+	/** Yellow octets that are sent through this tm node */
+	uint64_t yellow_octets;
+
+	/** Red packets that are sent through this tm node */
+	uint64_t red_pkts;
+
+	/** Red octets that are sent through this tm node */
+	uint64_t red_octets;
```

Comment:
Previous comments about renaming the struct members in `odp_tm_node_stats_capability_t `.